packages/workspace/src/command-line/utils.ts

```diff
@@ -199,6 +199,6 @@ export function getAffectedConfig(): NxAffectedConfig {
   const config = readNxJson();
 
   return {
-    defaultBase: config.affected?.defaultBase || 'master',
+    defaultBase: config.affected?.defaultBase || 'main',
```

Comment:
I'm not sure about changing this value :thinking: 

This will break everyone who is on older git, or has `master` as their main branch... and doesn't have it set in `nx.json` :thinking:

Reply:
1. See the first comment regarding older git.
2. Should we maybe add a migration to set it in existing repos w/o it set? It makes more sense to do this all at once to maintain consistency in docs.